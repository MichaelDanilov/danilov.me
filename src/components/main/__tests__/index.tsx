import React from 'react';
import renderer from 'react-test-renderer';
import Main from '..';

describe('Main', () => {
  it('english version renders correctly', () => {
    const tree = renderer.create(<Main lang="en" />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('russian version renders correctly', () => {
    const tree = renderer.create(<Main lang="ru" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
