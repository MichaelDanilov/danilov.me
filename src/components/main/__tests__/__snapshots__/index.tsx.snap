// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main english version renders correctly 1`] = `
<div
  className="sc-bdVaJa haSphr"
>
  <p
    className="sc-bwzfXH jfgEZP"
    dangerouslySetInnerHTML={
      Object {
        "__html": "<a href=\\"https://jamstack.org/\\" hreflang=\\"en-US\\">JAM-stack</a> developer.<br>Worked at <a href=\\"https://ningen.group/\\" hreflang=\\"en-US\\">Ningen Group</a>, <a href=\\"https://livetyping.com/en/\\" hreflang=\\"en-US\\">Live Typing</a>.<br>Now working at <a href=\\"https://yandex.com/\\" hreflang=\\"en-US\\">Yandex</a> in <a href=\\"https://www.kinopoisk.ru/\\" hreflang=\\"en-US\\">KinoPoisk</a> front-end team.<br>Lived in <a href=\\"https://en.wikipedia.org/wiki/Omsk\\" hreflang=\\"en-US\\">Omsk, Russia</a>. Now living in <a href=\\"https://en.wikipedia.org/wiki/Moscow\\" hreflang=\\"ru-RU\\">Moscow, Russia</a>",
      }
    }
  />
</div>
`;

exports[`Main russian version renders correctly 1`] = `
<div
  className="sc-bdVaJa haSphr"
>
  <p
    className="sc-bwzfXH jfgEZP"
    dangerouslySetInnerHTML={
      Object {
        "__html": "<a href=\\"https://jamstack.org/\\" hreflang=\\"en-US\\">JAM-stack</a> разработчик.<br>Работал в <a href=\\"https://ningen.group/\\" hreflang=\\"en-US\\">Ningen Group</a>, <a href=\\"https://livetyping.com/ru/\\" hreflang=\\"ru-RU\\">Лайв Тайпинге</a>.<br>Сейчас работаю в компании <a href=\\"https://yandex.ru/\\" hreflang=\\"ru-RU\\">Яндекс</a> в команде фронтенд-разработки <a href=\\"https://www.kinopoisk.ru/\\" hreflang=\\"ru-RU\\">КиноПоиска</a>.<br>Жил в <a href=\\"https://ru.wikipedia.org/wiki/Омск\\" hreflang=\\"ru-RU\\">Омске</a>. Сейчас в <a href=\\"https://ru.wikipedia.org/wiki/Москва\\" hreflang=\\"ru-RU\\">Москве</a>.",
      }
    }
  />
</div>
`;
